package.json

```diff
@@ -18,7 +18,7 @@
     "api"
   ],
   "dependencies": {
-    "unirest": "*"
```

Comment:
`^0.4.2` will pick up next minor version when package crossed v1 and next patch version when its before v1. So in this case if version `0.4.3` is released it will be picked automatically. 
(its because versions before v1 can have breaking changes)